### DISCUSSION TITLE
Is there a roadmap or at least a schedule for 1.0?

### DISCUSSION THREAD

**Discussion by jsoneaday** [leptos-rs/leptos#194] [General]
Love the project and very much want to use. Any ideas on first prod release? Thank you

**Comment by gbj** [comment]
Haha so w.r.t. a `1.0`: You're looking at `0.0` right now. `0.1.0` is in alpha. Because of the demands of semantic versioning, `1.0` would probably require `web-sys` and `wasm-bindgen` (the two big crates powering the Rust/Wasm ecosystem) to be at `1.0` (they've been at `0.3` and `0.2` respectively for most of the last few years) So, as is often the case for Rust crates, and especially for frontend web stuff, `1.0` is unlikely. (Yew, for example, is currently at `0.22`, having been around for about 5 years, compared to Leptos's 3 months.)

So, version numbers aside, what's needed for production-readiness, or what's the roadmap? Let's see... Finalizing `0.1.0` will be relatively big. We don't provide the server ourselves but offer integrations with Axum, Actix, etc., so there aren't big security or performance issues to worry about there. I'm sure there are bugs etc. to be fixed but the roadmap from this point is basically creating better docs and building up an ecosystem.
